scripts/bootstrapper/build.ps1

```diff
@@ -84,6 +88,10 @@ function Get-KoreBuild {
     $version = $version.TrimStart('version:').Trim()
     $korebuildPath = Join-Paths $DotNetHome ('buildtools', 'korebuild', $version)
 
+    if ($Reinstall -and (Test-Path $korebuildPath)) {
```

Comment:
Why do we need `-Reinstall` now?

Reply:
So the functional tests can force a re-install of a new build. I removed the setting that used a random, time-based build number for each build, so each build of build tools will be `2.0.0-rtm-t000`.

Reply:
Understood, so we don't need to propagate this change out to all the repos since it's test only, but when it does eventually copy that's fine.

Reply:
Yup. It's a non-essential change for other repos.